lib/pdf/reader/buffer.rb

```diff
@@ -52,7 +52,7 @@ class Buffer
     CR = "\r"
     LF = "\n"
     CRLF = "\r\n"
-    WHITE_SPACE = [LF, CR, ' ']
+    WHITE_SPACE = ["\n", "\r", ' ']
```

Comment:
sorbet ignored the type definition in the rbi file until I changed this to use literals and not reference the  earlier constants